recipe/meta.yaml

```diff
@@ -66,11 +79,18 @@ test:
     # show SIMD features (some failures occur depending on presence/absence of e.g. AVX512)
     # change to np.show_config() for v>=1.22.0, cf. numpy/numpy#19130
     - python -c "import numpy; numpy._pytesttester._show_numpy_info()"
-    # Run only fast tests for key modules on drone as it times out; otherwise set label='full' (note default: label='fast')
-    {% set label = "'fast'" if (aarch64 or ppc64le) else "'full'" %}
-    {% set tests = "['scipy.linalg', 'scipy.fft']" if (aarch64 or ppc64le) else "None" %}
-    # for signature, see https://github.com/scipy/scipy/blob/v1.7.0/scipy/_lib/_testutils.py#L27
-    - python -c "import scipy; scipy.test(verbose=2, label={{ label }}, tests={{ tests }}, extra_argv=['--durations=50'])"
```

Comment:
FWIW we were able to get this to work in `pywavelets` by wrapping it in `sys.exit(not ...)`

https://github.com/conda-forge/pywavelets-feedstock/pull/43/commits/754b671314413159fd7a33e98dea4cb381067a4a

Reply:
Yeah, that's what I was wondering about [above](https://github.com/conda-forge/scipy-feedstock/pull/195#issuecomment-962684716). Added a commit to wrap `sys.exit` (while trying to keep line length readable).